scripts/backend_test_shards.json

```diff
@@ -1,6 +1,7 @@
 {
     "1": [
         "core.jobs.transforms.job_result_transforms_test",
+        "core.jobs.transforms.results_transforms_test",
```

Comment:
Shard 1 is now [taking longer than 30 mins](https://github.com/oppia/oppia/actions/runs/3506252724/jobs/5873165813). Please add this test to a different shard so that all shards run in <30 mins to keep CI balanced. You can read more about this policy in the backend test wiki page

Reply:
Done